lib/utils/testLauncherHelper.js

```diff
@@ -389,6 +389,7 @@ function getChildOptions(deviceId, deviceAccessToken, appConfigId, isDebugMode,
 		[envVars.SUITEST_APP_CONFIG_ID]: appConfigId,
 		[envVars.SUITEST_DEBUG_MODE]: isDebugMode ? 'yes' : 'no',
 		[envVars.SUITEST_CONFIG_LOG_LEVEL]: config.logLevel,
+		[envVars.SUITEST_CONFIG_DEFAULT_TIMEOUT]: config.defaultTimeout,
```

Comment:
Having something called testLauncherHelper.js used outside test environment can cause confusion😕 I know this is a testing tool, but still...😵 maybe should be renamed to suitestLauncherHelper or something(in a new task). Just a thought.